### PR TITLE
correct ORNL_19pin documentation 

### PR DESCRIPTION
**Pull request by kyriv-lab**
This PR is associated with an issue: #29484
The documentation of SCM.

## Reason
The regression test related with the ORNL-19 pin validation, was not representative of the actual problem.
This was confusing to users trying to run the validation cases listed in the documentation.

## Design
Use the low flow input case to create a regression test and correct the documentation to reflect that. This requires re-golding.

## Impact
The users can now edit the regression test file to reproduce the other cases mentioned in the documentation by editing the boundary conditions.

### CODE REVIEW

File: modules/subchannel/doc/content/modules/subchannel/v&v/ornl_19_pin.md [hunk 1]
```diff
@@ -60,15 +60,19 @@ These configurations are summarized in [cases].
 
 ### General parameters
 
-The general parameters on the experimental conditions are described here below.
-They set up the boundary conditions for the high-flow-rate test case.
+The general parameters on the experimental conditions are described here. The purpose is to define inlet conditions: Pressure, Temperature and mass-flux. For the low-flow-rate test case presented here, we have:
 
 ```language=bash
```
Comment (GiudGiud) [inline modules/subchannel/doc/content/modules/subchannel/v&v/ornl_19_pin.md]: use an input file listing instead

Comment (kyriv-lab) [reply modules/subchannel/doc/content/modules/subchannel/v&v/ornl_19_pin.md]: like: !include inlet_boundary_conditions.i

Comment (GiudGiud) [reply modules/subchannel/doc/content/modules/subchannel/v&v/ornl_19_pin.md]: !listing ornl_... .i start=this line end=that line

Comment (GiudGiud) [reply modules/subchannel/doc/content/modules/subchannel/v&v/ornl_19_pin.md]: it's not the line number you put there, it's the content of the line

File: modules/subchannel/doc/content/modules/subchannel/v&v/ornl_19_pin.md [hunk 2]
```diff
@@ -60,17 +60,9 @@ These configurations are summarized in [cases].
...
 
-```language=bash
-
- T_in = 588.5
- flow_area = 0.0004980799633447909 #m2
- mass_flux_in = ${fparse 55*3.78541/10/60/flow_area}  # [1e+6 kg/m^2-hour] turns into kg/m^2-sec
- P_out = 2.0e5 # Pa
-
-```
+!listing /validation/ORNL_19_pin/test_ORNL_19.i start=T_in end=P_out
```
Comment (GiudGiud) [inline modules/subchannel/doc/content/modules/subchannel/v&v/ornl_19_pin.md]:
```suggestion
!listing /validation/ORNL_19_pin/test_ORNL_19.i start=T_in end=P_out include-end=true
```

**Review by GiudGiud — CHANGES_REQUESTED** [review]

**Comment by GiudGiud** [comment]
let's squash this in pls

**Review by GiudGiud — APPROVED** [review]
@tanoret could you please do the technical review. thank you!

**Review by tanoret — APPROVED** [review]
The results seem good and the new volumetric flow rate is consistent with the specifications of the ORNL cases. Good catch!
